src/plugins/lua/bindings/lua_ImGuiCustom.cpp

```diff
@@ -99,6 +102,54 @@ void RegisterBindings_ImGuiCustom(sol::table& ImGui)
 			[](mq::imgui::ConsoleWidget* pThis, const ImVec4& col, std::string_view text) { pThis->AppendText(text, MQColor(col), false); }
 		)
 	);
-}
 
+	// Widgets: ZepInputBox
+	ImGui.new_usertype<mq::imgui::ImGuiZepEditor>(
+	"ZepEditor", sol::factories(&mq::imgui::ImGuiZepEditor::Create),
```

Comment:
I'm not sure we should expose the implementation detail of "zep" into lua space. We should probably just have a "TextEditor". The user doesn't need to know the tech behind it

Reply:
I think I agree with this. I told derple that a lot of the naming things are TBD once I have a chance to review and see everything that is involved.

Reply:
Upated